.github/workflows/test.yaml

```diff
@@ -13,26 +13,36 @@ jobs:
       matrix:
         go-version: [1.16.x]
     runs-on: ubuntu-latest
-    services:
-      hardhat:
-        image: smartcontract/hardhat-network
-        ports:
-          - 8545:8545
-        options: >-
-          --health-cmd "curl -X POST --data '{"jsonrpc":"2.0","method":"net_listening","params":[],"id":31337}' http://localhost:8545"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
+    env:
+      KUBECONFIG: ${{ secrets.KUBECONFIG }}
```

Comment:
Looks like you can't store the raw config in the env, only the path. Write this secret to file then specify the file path in this env.